pkg/relay/register.go

```diff
@@ -0,0 +1,249 @@
+package relay
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"sync/atomic"
+
+	"github.com/blocknative/dreamboat/pkg/structs"
+	"github.com/flashbots/go-boost-utils/types"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+const (
+	ResponseTypeVerify = iota
+	ResponseTypeOthers
+	ResponseTypeStored
+)
+
+const (
+	ResponseQueueSubmit = iota
+	ResponseQueueRegister
+	ResponseQueueOther
+)
+
+type Setter interface {
+	Set(k string, value uint64)
+}
+
+type Getter interface {
+	Get(k string) (value uint64, ok bool)
+}
+
+type SVRReq struct {
```

Comment:
Could we rename this and the structs below to something more descriptive?

Reply:
Mmm I understand why you want to keeep it short. But VS could be anything. I think we should use VerifyReq, which is only one letter more than StoreReq that is below